dbt/context/common.py

```diff
@@ -279,6 +280,7 @@ def generate(model, project, flat_graph, provider=None):
         "model": model,
         "post_hooks": post_hooks,
         "pre_hooks": pre_hooks,
+        "pytz": pytz,
```

Comment:
`pytz` is the first module that we're adding to the dbt context, but conceivably we'd want to add many more like `datetime`, `collections` etc. That makes me think we should namespace all of these modules under a `libraries` or `modules` or `imports` name, so the namespace doesn't get super polluted.

```
"modules": {
  "pytz" : pytz,
  "datetime": datetime,
  ....
}
```

@mturzanska  and @cmcarthur what do you think about that?

Reply:
namespacing these is a good idea. `modules` seems like a perfectly good name for that namespace.

let's just make sure the syntax remains nice, i.e. (`modules.pytz.timezone("America/New_York")`). @mturzanska if that syntax doesn't work after updating this, you can make `modules` an `AttrDict`:

```
...
"modules": AttrDict({
  "pytz": pytz
})
...
```

should do the trick.